README.md

```diff
@@ -15,21 +15,38 @@ This document outlines the procedure for changes to the constitution held in thi
     1. HackSoc will be referred to as the 'Society'.
     2. The constitution of the Society, as laid out in [Constitution.md](./Constitution.md), will be referred to as the 'Constitution'.
 
-## 2. Changes to the Constitution
+## 2. General Rules
 
-1. All changes to the Constitution MUST be made via a Pull Request.
-2. Pull Requests for changes to the Constitution MUST be merged following a successful vote at a General Meeting of the Society.
-3. 'Housekeeping' changes MAY be made on the conditions that said changes:
-    1. MUST not affect the meaning of the Constitution (though they MAY affect the format and/or style);
-    2. MUST be made via a Pull Request as per 2.1, for which the following restrictions apply:
+1. All changes to this repository MUST be made via a Pull Request.
+2. 'Housekeeping' changes MAY be made on the conditions that said changes:
+    1. MUST NOT affect the meaning of the Constitution or its appendices (though they MAY affect the format and/or style);
+    2. MUST be made via a Pull Request as per §2.1, for which the following restrictions apply:
```

Comment:
for the sake of keeping things consistent with the rest of the constitution, at least for the time being:

```suggestion
    2. MUST be made via a Pull Request as per 2.1, for which the following restrictions apply:
```

(I would personally consider adding `§` to section numbers later to be a housekeeping change that would not require an AGM/EGM)